docs/index.rst

```diff
@@ -77,12 +77,15 @@ and some of the core features of Read the Docs.
 * **Overview of core features**:
   :doc:`Incoming webhooks </webhooks>` |
   :doc:`/custom_domains` |
+  :doc:`/canonical` |
```

Comment:
I feel like this file could have a better name, almost certainly. That's true of most of our docs tho, perhaps for a later time to reduce redirect overhead.

Reply:
Let's address that in a future PR, we have enough redirections to create on this pass already...